gradlew

```diff
@@ -1,7 +1,7 @@
-#!/usr/bin/env sh
+#!/bin/sh
 
 #
-# Copyright 2015 the original author or authors.
+# Copyright © 2015-2021 the original authors.
```

Comment:
Just for my understanding, should this be 2025 or is this script auto-generated? (https://www.apache.org/licenses/LICENSE-2.0#apply)

Reply:
This is auto generated. I can fix this manually